### DISCUSSION TITLE
Demo Discovering and Using OPC UA Thermometers

### DISCUSSION THREAD

**Discussion by PE-29012018** [project-akri/akri#723] [Q&A]
Hello everyone, 

I am currently testing the demo ‘Recognising and using OPC UA thermometers’. 
I can set up the OPC UA servers (without security) without any problems. 
I can also install Akri as in the example. I can also see the required CRDs.
When I run ‘kubectl get akric’, I also see akri-opcua-monitoring. 
However, as soon as I run ‘kubectl get akrii’ as required by the next step, I get no output. 

What could be the reason for this?

**Comment by kate-goldenring** [comment]
It sounds like the discovery handler is not discovering your servers. Can you share the logs of the OPC UA discovery handler pod?
